README.ru.md

```diff
@@ -104,7 +104,8 @@ Backend сервиса реализован на Go, frontend на TypeScript, 
 
 * `mysql` - реквизиты пользователя хранятся в таблице MySQL. Структура таблицы и метод хэширования пароля унаследованы и совместимы с модулем `mod_auth_mysql` для Apache httpd.
 
-* `header` - аутентификация пользователя осуществляется механизмами reverse proxy, развернутого перед *onlineconf-admin*, который в случае успеха передает дополнительный HTTP заголовок с именем аутентифицированного пользователя (плюс, опционально, подпись).
+* `header` - аутентификация выполняется обратным прокси-сервером, развернутым перед onlineconf-admin. Аутентификация этого типа требует параметра конфигурации `behind_reverse_proxy` и одного дополнительного параметра конфигурации `header`, который устанавливает имя заголовка с `username`. Например, если вы используете oauth2-proxy, имя заголовка с именем пользователя — «X-Forwarded-Preferred-Username».
+Как это работает. В случае успеха обратный прокси добавляет один обязательный HTTP-заголовок с именем пользователя и один дополнительный HTTP-заголовок с подписью. По умолчанию имя заголовка с именем пользователя — «X-Username». Заголовок, содержащий подпись, использует то же имя, но с постфиксом `-Sign` (по умолчанию `X-Username-Sign`). Подпись рассчитывается как хешированная строка MD5, состоящая из необходимого значения заголовка (по умолчанию `X-Username`), `remote_addr` и секретного параметра конфигурации onlineconf-admin.
```

Comment:
No problem. Up to you.
But small corrections in RU translation "передачи имя пользователя" -> "передачи имени пользователя"

Reply:
* "HTTP header" is just a generic text, not something special that requires monospace.
* "onlineconf-admin" in this paragraph (RU) should be used consistently with or without backquotes. Throughout  this README `onlineconf-admin` in most cases is used then speaking about program (binary) and *onlineconf-admin* then about subsystem.

Reply:
It's a very difficult for my. I don't understand what about "HTTP header" at all. And I think what I fix uses `onlineconf-admin`

P.S. this "PR" takes to lot of my time and become not reasonably for my. If you can make small corrections instead of spending a lot of time explaining it in the comments, it will make the whole process easier without long discussions.

Reply:
I see, you understood about "HTTP header" and fixed it)

The same thing about spending a lot of time equally applies to me. I'd prefer to merge requests at the first try, but this is not always possible. I can't say this code review was totally useless. Maybe some things here are a little bit opinionated but code reviews are used not only to fix errors but to allow authors to achieve agreement on how it must work or look like. This project is rather stable and I like idea to merge high quality code into it if possible.